aws_iam_access_key.tf

```diff
@@ -14,7 +14,8 @@ resource "aws_iam_user" "NewUser" {
 }
 
 resource "aws_iam_access_key" "NewUserAccessKey" {
-  user    = "${aws_iam_user.NewUser.name}"
+  user   = "${aws_iam_user.NewUser.name}"
+  status = "Inactive"
 }
```

Comment:
**Accurics remediation**
 
**[MEDIUM]: IAM user with permanent programmatic access detected**
 
**Vulnerability**
Ensure that there are no exposed Amazon IAM access keys in order to protect your AWS resources against unapproved access
 
**Remediation**
Ensure that there are no exposed Amazon IAM access keys in order to protect your AWS resources against unapproved access

source: accurics-